shared/message/support/FieldDescription.proto

```diff
@@ -72,6 +72,10 @@ message FieldDescription {
         double center_circle_diameter = 16;
 
         double border_strip_min_width = 17;
+
+        double penalty_area_length = 18;
+
+        double penalty_area_width = 19;
     }
```

Comment:
you don't need to do all of them, but probably add in documentation directives for the ones you added at least.